src/components/FeaturePanel/renderers/openingHours/index.ts

```diff
@@ -0,0 +1,10 @@
+import { parseComplexOpeningHours } from './complex';
+import { parseSimpleOpeningHours } from './simple';
+
+export const parseOpeningHours = (value: string) => {
+  try {
+    return parseComplexOpeningHours(value);
+  } catch {
+    return parseSimpleOpeningHours(value);
```

Comment:
Do you expect this exception more often? I would say we can discard the `simple-opening-hours` lib altogether. If there is an exception, just return `null` and log the exception to sentry.. what do you think?

If we do this it can also simplify the `<OpeningHoursRenderer>` component to tailor it exactly to the new lib needs.